flows/src/test/java/com/softwaremill/jox/flows/FlowIOTest.java

```diff
@@ -395,8 +395,8 @@ void runToFile_throwExceptionIfFileCannotBeOpened() {
         Path path = folder.resolve("not-existing-file.txt");
         var source = Flows.fromByteArrays(new byte[0]);
 
-        // when & then
-        assertThrows(NoSuchFileException.class, () -> source.runToFile(path));
+        // when & then  Linux: NoSuchFileException
```

Comment:
so on other OSes there's a different exception? I'm not sure I understand the change :)